assets/Product.wxs

```diff
@@ -60,6 +60,30 @@
 
     <!--We need to show EULA, and provide option to customize download location-->        
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER" />
+	
+    <!-- Prerequisites check for Windows Universal C time and Visual Studio 2015 C++ redistributables -->
+    <Property Id="UCRTINSTALLED" Secure="yes">
+      <DirectorySearch Id="Windows_System32" Path="[WindowsFolder]System32" Depth="0">
+        <FileSearch Name="ucrtbase.dll"/>
+      </DirectorySearch>
+    </Property>	
+    <Property Id="VCREDISTINSTALLEDX64" Secure="yes">
+      <RegistrySearch Id="VCRedist_RegSearch_x64" Root="HKLM" Name="Version" Type="raw" Win64="yes"
+                      Key="SOFTWARE\Microsoft\DevDiv\VC\Servicing\14.0\RuntimeMinimum"/>
+    </Property>
+    <Property Id="VCREDISTINSTALLEDX86" Secure="yes">
+      <RegistrySearch Id="VCRedist_RegSearch_x86" Root="HKLM" Name="Version" Type="raw" Win64="no"
+                      Key="SOFTWARE\Microsoft\DevDiv\VC\Servicing\14.0\RuntimeMinimum"/>
+    </Property>
+    <Condition Message="$(env.ProductName) requires the Universal C Runtime to be installed. You can download it here: https://www.microsoft.com/en-us/download/details.aspx?id=50410">
+      <![CDATA[Installed OR UCRTINSTALLED]]>
+    </Condition>
+    <Condition Message="$(env.ProductName) requires the Visual Studio C++ 2015 x64 redistributables to be installed. You can download it here: https://www.microsoft.com/en-gb/download/details.aspx?id=48145">
```

Comment:
It's close. I overlooked this the first time. Use a link without the language specifier so that it uses the browser's default language. For example:
https://www.microsoft.com/download/details.aspx?id=48145 automatically redirects to en-us for me.

This should also be fixed in the other files as well.